apps/web/src/server/views/lpa/check-and-confirm.njk

```diff
@@ -0,0 +1,48 @@
+{% extends "_layouts/layout-base.njk" %}
+
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block header %}
+	{{ govukHeader({
+		homepageUrl: "https://www.gov.uk/",
+		serviceName: "Appeal a planning decision",
+		serviceUrl: "/lpa",
+		containerClasses: "govuk-width-container"
+	}) }}
+{% endblock %}
+
+{% block content %}
+	{{ super() }}
+
+	<h1 class="govuk-heading-xl">
+		Review questionnaire
+	</h1>
+
+	<h2 class="govuk-heading-l">
+		Check and confirm
+	</h2>
+
+	{{ govukSummaryList({
+		rows: [
+			{ key: { text: "Review outcome" }, value: { text: reviewOutcomeLabel } },
+			{ key: { text: "Appeal reference" }, value: { text: questionnaireData.AppealReference } },
+			{ key: { text: "Appeal site" }, value: { html: questionnaireData.AppealSiteHtml } },
+			{ key: { text: "Local planning department" }, value: { text: questionnaireData.LocalPlanningDepartment } }
+		]
+	}) }}
+
+	{%- if reviewOutcome === 'complete' -%}
+		{{ govukWarningText({
+			iconFallbackText: "Warning",
+			text: "Confirming this questionnaire as complete makes the appeal visible to inspectors."
+		}) }}
+	{%- endif -%}
+
+	<div class="govuk-button-group">
+		{{ govukButton({ text: 'Confirm outcome' | default('Confirm outcome') }) }}
```

Comment:
why do we need the` | default` filter? Isn't that just for when the initial text is undefined and it falls back to the default value?

Reply:
This was a result of copying templates where the text is passed in as a variable, I've removed the defaults from the LPA templates now